setupTest.js

```diff
@@ -3,3 +3,12 @@
 process.env.CHOKIDAR_USEPOLLING = true;
 
 jest.setTimeout(140000);
+
+if (expect.getState().testPath.includes("test/e2e")) {
+  afterEach(() => {
+    // We add listeners to signals when creating a new Server instance, but have no cleanup method
+    // So ensure they are removed after each run to prevent EventEmitter memory leak warnings
+    process.removeAllListeners("SIGINT");
+    process.removeAllListeners("SIGTERM");
+  });
+}
```

Comment:
hm, it is because we attach to `process`? Maybe we should do it in server in close

Reply:
Yes, we should move it to the `stop` method.

Reply:
Let's do it